packages/server/src/sdk/app/rows/utils.ts

```diff
@@ -1,12 +1,12 @@
 import cloneDeep from "lodash/cloneDeep"
 import validateJs from "validate.js"
-import { FieldType, Row, Table, TableSchema } from "@budibase/types"
+import { FieldType, QueryJson, Row, Table, TableSchema } from "@budibase/types"
 import { makeExternalQuery } from "../../../integrations/base/query"
 import { Format } from "../../../api/controllers/view/exporters"
 import sdk from "../.."
 import { isRelationshipColumn } from "../../../db/utils"
 
-export async function getDatasourceAndQuery(json: any) {
+export async function getDatasourceAndQuery(json: QueryJson) {
   const datasourceId = json.endpoint.datasourceId
   const datasource = await sdk.datasources.get(datasourceId)
   return makeExternalQuery(datasource, json)
```

Comment:
Would be nice to give this a type other than `any` if possible.

Reply:
That is what this change does.

Reply:
Sorry, I mean the return type. My editor still says it's `Promise<any>` unless I'm missing something.